servers/republik/modules/crowdfundings/lib/generateMemberships.js

```diff
@@ -68,6 +76,12 @@ module.exports = async (pledgeId, pgdb, t, logger = console) => {
             membership
           }
         } else {
+          if (
+            ['ABO_BENEFACTOR', 'ABO', 'ABO_REDUCED'].includes(membershipType.name) &&
```

Comment:
`ABO_BENEFACTOR`  and `ABO_REDUCED` are non-existant "membership types". Terms exists as made-up terms for CSV export to distinguish data. There is probably just a `BENEFACTOR_ABO`, but no reduced version of it.

Checking up on that.